src/Matterhorn/State/Attachments.hs

```diff
@@ -68,9 +68,8 @@ attachFileByPath txtPath = do
     let strPath = unpack txtPath
     fileInfo <- liftIO $ FB.getFileInfo strPath strPath
     case FB.fileInfoFileStatus fileInfo of
-        Left _ -> do
-            mhLog LogError $ "Failed to obtain FileInfo for file: " <> txtPath
-            mhError $ BadFile txtPath
+        Left e -> do
+            mhError $ BadAttachmentPath (toException e) "Unable to stat the requested file.  Check that it exists and has proper permissions"
```

Comment:
I'm concerned that the word "stat" here won't be well-known to users. Could we say something similar, like "Unable to attach the requested file. Check that it exists and has proper permissions."?

Reply:
This should be addressed now